### ISSUE TITLE
[BUG] Kalenderdarstellung Wochenansicht - Termine desselben Sachbearbeiters

### ISSUE THREAD

**Issue by iradraconis** [jlawyerorg/j-lawyer-org#2033]
Die beiden grünen Termine haben eine Lücke von 5 Minuten, werden aber nicht übereinander angezeigt. Bei dem zweiten Beispiel liegen die Termine exakt "aneinander".

![Bildschirmfoto vom 2023-08-14 12-28-24](https://github.com/jlawyerorg/j-lawyer-org/assets/71747937/3ae868ee-4b51-41bd-8761-d828d7816cee)

![grafik](https://github.com/jlawyerorg/j-lawyer-org/assets/71747937/52977c40-d329-49a8-a37c-f7f3480ffc51)

**Comment by iradraconis** [comment]
Machen wir irgendwas falsch? :)

![Bildschirmfoto vom 2023-09-26 20-31-29](https://github.com/jlawyerorg/j-lawyer-org/assets/71747937/b4309caa-c18f-4a11-b735-22fe6612ddf9)

**Comment by j-dimension** [comment]
Nee, das Rendering ist suboptimal. Ticket liegt aber schon in der 2.5.

**Comment by iradraconis** [comment]
In den meisten Konstellationen ist die Darstellung jetzt besser.

Aus: 

<img width="714" height="782" alt="Image" src="https://github.com/user-attachments/assets/1ce8454f-0c2f-40d3-9fb6-79e4b73b775b" />


wird:

<img width="714" height="782" alt="Image" src="https://github.com/user-attachments/assets/8bec3a6e-0311-4847-bda4-b6095ee599c2" />

Und noch ein Beispiel mit einer überlappenden Zeit:

<img width="714" height="782" alt="Image" src="https://github.com/user-attachments/assets/c79602ef-3891-43a9-8268-606a309ec88c" />
